src/configur.sh

```diff
@@ -196,11 +196,10 @@ gen_watcom ()
   ../util/linux/mkmake -w -o watcom_3.mak -d build/watcom/small32 makefile.all WATCOM SMALL32
 
   ../util/linux/mkdep -s.obj -p\$\(OBJDIR\)/ *.[ch] > build/watcom/watt32.dep
-
   echo "neterr.c: build/watcom/syserr.c" >> build/watcom/watt32.dep
-# these hacks won't work because errnos.c relies on being compiled as a target-exe.
-# ../util/wc_err -s > build/watcom/syserr.c
-# ../util/wc_err -e > ../inc/sys/watcom.err
+
+  # these require dosemu be install
+  dosemu -dumb -c "../dosemu.bat"
```

Comment:
Why not generate `$(TEMP)/dosemu.bat`? And with a proper comment. I has really no place for DOS/Win32.

Reply:
It uses implicit relative path mapping that all paths are relative to bat file location. watt-32 root is implicitly map to D:  drive. I can create *.bat file on the fly, but two dosbox mapped drives will be necessary and we will need to know absolut path for each. Now all is relative to dosbox.bat file location in watt-32 tree.